lib/irslackd.js

```diff
@@ -506,6 +506,12 @@ class Irslackd {
   async onSlackReady(ircUser, event) {
     // Send MOTD
     this.ircd.write(ircUser.socket, 'irslackd', '001', [ ircUser.ircNick, 'irslackd' ]);
+    // Send RPL_ISUPPORT
+    this.ircd.write(ircUser.socket, 'irslackd', '005', [
+      ircUser.ircNick,
+      'CHANTYPES=#',
```

Comment:
There's also a debug channel that uses the `&` prefix. Would that also belong here?

Reply:
Yes